WordImpress/Give
# feat(admin-plugins): add a popup survey upon plugin deactivation #648

Closes #648 

## Description
This PR implements the deactivation survey feature.
The bar graph is implemented using [Chart.js](http://chartjs.org/)

## How to test this feature?
1. Create a **client website** (client.give.com) which has the Give Donation Plugin.
2. Create a **survey website** (survey.give.com) which has the [deactivation-survey](https://github.com/WordImpress/Give-Deactivation-Survey) plugin. This website will be the endpoint to receive and store survey data.
3. Replace [this](https://github.com/WordImpress/Give/blob/issue-648/includes/admin/plugins.php#L575) URL with `http://survey.give.com/wp-json/give/v2/survey/`
4. Make sure the respective plugins are activated on both the sites.
5. Watch the video linked below.

## Video
https://www.useloom.com/share/7921b7d8039f43b0a1f0a59470ed540b

## Checklist:
